# pandas.errors.MergeError

exceptionpandas.errors.MergeError[source]

    
Error raised when problems arise during merging due to problems with input
data. Subclass of ValueError.

© 2008–2022, AQR Capital Management, LLC, Lambda Foundry, Inc. and PyData
Development Team  
Licensed under the 3-clause BSD License.  
https://pandas.pydata.org/pandas-
docs/version/1.4.0/reference/api/pandas.errors.MergeError.html

